src/components/About/About.js

```diff
@@ -97,6 +90,72 @@ const About = (props) => {
     );
   }
 
+  function renderWarningBanner(interfacesList) {
+    const modulesArray = Object.keys(props.modules).map(key => props.modules[key]);
+    const missingModules = _.flattenDeep(modulesArray.map(module => {
+      return module.map(item => {
+        const okapiInterfaces = item.okapiInterfaces;
+        return okapiInterfaces && Object.keys(okapiInterfaces).map(key => {
+          const required = okapiInterfaces[key];
+          const available = interfacesList[key];
+
+          if (!available) {
+            return `${key} ${required}`;
+          } else {
+            return null;
+          }
+        });
+      });
+    }));
+
+    const incompatibleModule = _.flattenDeep(modulesArray.map(module => {
```

Comment:
and seems all these maps are copy paste of missing modules, only difference is condition